examples/js/interactive/SelectionBox.js

```diff
@@ -65,7 +75,7 @@ THREE.SelectionBox = ( function () {
 			endPoint.x = Math.max( startPoint.x, endPoint.x );
 			endPoint.y = Math.min( startPoint.y, endPoint.y );
 
-			vecNear.copy( this.camera.position );
+			vecNear.copy( this.camera.getWorldPosition() );
```

Comment:
`getWorldPosition()` is a bit resource intensive if frequently called. Try it with:
```js
vecNear.setFromMatrixPosition( this.camera.matrixWorld );
```

Reply:
Updated.